doc/diffusion.bib

```diff
@@ -319,14 +319,14 @@ @Article{Carlsson2009
 }
 
 @Article{Avram2008NMRBiomed,
-  Author         = {Avram, Liat and \�{O} zarslan, Evren and Assaf, Yaniv
+  Author         = {Avram, Liat and \�{O} zarslan, Evren and Assaf, Yaniv
                    and Bar-Shir, Amnon and Cohen, Yoram and Basser, Peter
                    J.},
   Title          = {Three-dimensional water diffusion in impermeable
                    cylindrical tubes: theory versus experiments},
   Journal        = {NMR IN BIOMEDICINE},
   Volume         = {21},
-  Pages          = {888�898},
+  Pages          = {888�898},
```

Comment:
Similar to the above, but in this case the hyphen does exist in BibTeX; usually BibTeX is smart enough to change it to the en dash for page ranges.

Reply:
Removed existing entries were removed in PR #3308. Resolving this.